pkg/controller/clone-controller.go

```diff
@@ -627,6 +627,7 @@ func MakeCloneSourcePodSpec(sourceVolumeMode corev1.PersistentVolumeMode, image,
 					VolumeSource: corev1.VolumeSource{
 						PersistentVolumeClaim: &corev1.PersistentVolumeClaimVolumeSource{
 							ClaimName: sourcePvcName,
+							ReadOnly:  true,
```

Comment:
Can you set this to true only for filesystem volumes? We have had provisioners that didn't like read only block volumes. Around line 712 we have a check that sets readonly on the volumeMount already.

Reply:
this came up last time when we faced an issue with read only in the host assisted path
the conclusion was that we'll always prefer compatibility in the host assisted case, so imposing a requirement here (even if only for fs) is risky business imo

Reply:
I recall getting permission errors with this setting on ceph filesystem volumes.  Was hoping that readOnly set here would be sufficient:  https://github.com/kubevirt/containerized-data-importer/blob/3426b9c2a939e0da69a7be72dee37288b9676cd3/pkg/controller/clone-controller.go#L712

Reply:
you're right
https://storage.googleapis.com/kubevirt-prow/pr-logs/pull/kubevirt_containerized-data-importer/3347/pull-containerized-data-importer-e2e-ceph/1813599133605302272/artifacts/1/pods/serial_2_cdi-e2e-tests-cloner-hk8fs_b17707b3-105a-45e3-af55-a162ff6b5497-source-pod-cdi-clone-source_previous.log

Reply:
saw that, looking for other directions.